### ISSUE TITLE
Consider moving constants to the top of the AirScript file

### ISSUE THREAD

**Issue by tohrnii** [0xPolygonMiden/air-script#89]
An alternative structure proposed by @grjte for constants is to move it outside source section to the top of the AirScript file.
The syntax for declaring constants in this case could look like:
```
def ExampleAir

const A = 1
const B = 0

trace_columns:
...
```

**Comment by bobbinth** [comment]
I like this approach!

**Comment by grjte** [comment]
Since we're agreed, I'm adding this to #77 as a "nice to have"

**Comment by tohrnii** [comment]
Closed by #103.
